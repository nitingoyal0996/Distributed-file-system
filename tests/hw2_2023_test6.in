create f1
append f1 SLICE_TEST_012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789_SLICE_TEST
cat f1
slice f1 1 260
cat f1
slice f1 1 5
cat f1
exit

