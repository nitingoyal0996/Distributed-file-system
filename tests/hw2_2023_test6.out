#### File system information:
Number of blocks          : 256
Block size (Bytes)        : 256
Number of inodes          : 32
inode size (Bytes)        : 32
inodes per block          : 8
Free bitmap offset        : 2
Free bitmap size (blocks) : 1
Inode table offset        : 3
Inode table size (blocks) : 4
Max blocks per file       : 6
Data blocks offset        : 7
Data block size (blocks)  : 249
Raw block layer layout: (B: boot, S: superblock, F: free bitmap, I: inode, D: data
0123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345
BSFIIIIDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD
[cwd=0]%
[cwd=0]%
Successfully appended 292 bytes.
[cwd=0]%
SLICE_TEST_012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789_SLICE_TEST
[cwd=0]%
[cwd=0]%
LICE_TEST_0123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789
[cwd=0]%
[cwd=0]%
ICE_T
[cwd=0]%
